src/database/DataTypes/Transaction.js

```diff
@@ -395,6 +395,7 @@ export class Transaction extends Realm.Object {
       itemBatch.batch = this.adjustBatchName(batch);
       itemBatch.costPrice = packedToOneCostPrice;
       itemBatch.sellPrice = packedToOneSellPrice;
+      itemBatch.supplier = this.otherParty;
```

Comment:
should check isIncomingInvoice just in case:
https://github.com/openmsupply/mobile/blob/d44edcda7c7da26b1b58006a01c18d07f7fc31b0/src/pages/CustomerInvoicesPage.js#L85-L91

Reply:
hmm, do you mean the getter `Transaction.isIncoming` - This only checks for the `supplier_invoice` type, which is done in the query, or do you mean something else?

Reply:
hmm, i was just looking at the code:
```
    const newNumberOfPacks = isIncomingInvoice
        ? itemBatch.numberOfPacks + packedToOneQuantity
        : itemBatch.numberOfPacks - packedToOneQuantity;
```

it implies this method is used for incoming and outgoing transactions, not sure if that's the case, but if it is then may be setting supplier on the batch to something undesirable. Would inventory adjustments use this method also ?